.github/workflows/static-analysis.yml

```
@@ -0,0 +1,49 @@
+on:
+  - push
+
+name: Run PHPStan checks
+
+jobs:
+  mutation:
+    name: PHPStan ${{ matrix.php }}-${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+
+        php:
+          - "8.1"
+          - "8.2"
+          - "8.3"
```

```suggestion
          - "8.3"
          - "8.4"
```
PHP 8.4 support was added when branch 4.0 was created.